homepage.php

```diff
@@ -13,9 +13,6 @@
 	</head>
 <body>
 	<div class ="grid-container">
-		<div class="img_1">
```

Comment:
In the layout 2 I decided to make the nav bar on the top then the img will be underneath so I remove this can out it in the bottom so it make more sense